rotkehlchen/globaldb/upgrades/manager.py

```diff
@@ -28,6 +29,10 @@
         from_version=3,
         function=migrate_to_v4,
     ),
+    UpgradeRecord(
+        from_version=4,
+        function=migrate_to_v5,
+    ),
```

Comment:
This shouldn't be in the globaldb. It should be store in the user's db in the table we have for rpc nodes.